src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -342,6 +346,8 @@ private void Pane_Loaded(object sender, RoutedEventArgs e)
 		{
 			((UIElement)sender).GotFocus += Pane_GotFocus;
 			((UIElement)sender).RightTapped += Pane_RightTapped;
+
+			PaneLeft.RootGrid.Translation = new System.Numerics.Vector3(0, 0, 8);
```

Comment:
@0x5bfa is this needed, or does GotFocus always get called on startup?

Reply:
No, it looks like not called on the startup, therefore, added.
Do you think it doesn’t have to be focused on the startup?

Reply:
What if we assign the default translation from XAML?

Reply:
We'll still have to adjust it when the focus changes, but at least we'll be able to remove this line and clean up the code behind a bit.

Reply:
> What if we assign the default translation from XAML?

Thought the same. Code will be simplified but have to call VisualState.GoToState(this, “LeftShadowOn”, true);
However this makes me one more hurdle to refactor the pane.

Reply:
Let's keep as is for now